src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -256,16 +225,22 @@ public String getUrlName() {
     public void doTestSubmitBuildRef(StaplerRequest req, StaplerResponse rsp,
             @QueryParameter(required = true) String imageId,
             @QueryParameter(required = true) String jobName) throws IOException, ServletException {
-        Run latest = Jenkins.getInstance().getItem(jobName, Jenkins.getInstance(), 
-                AbstractProject.class).getLastBuild();
```

Comment:
Agreed. We still use this nondocumented method for stubbing till I submit a fix for JENKINS-28658